proposals/2022-03-decrease-voting-period/proposal.json

```diff
@@ -0,0 +1,11 @@
+{
+    "title": "Decrease Voting Period",
+    "description": "This is a parameter change proposal decreasing the voting period for all governance proposals from 14 days to 7 days. \n\nMore information can be found in the long-form proposal: https://github.com/regen-network/governance/tree/main/proposals/2022-03-decrease-voting-period",
```

Comment:
By using `main`, there's a risk that the proposal changes between its submission and the voting period end. I would prefer using a hash.

Reply:
The revision history can be viewed in git history. I'm not entirely opposed to using a git hash but we've been using `main` to allow for minor revisions such as updating links. In the last proposal, one of the dates were incorrect, which could have (or should have) been prevented with a better review process. I'm leaning in favor of using `main` given the git history can be viewed on all these files but also not strongly opposed to using a git hash.